diff --git a/node_modules/@openreplay/tracker-assist/lib/Assist.d.ts b/node_modules/@openreplay/tracker-assist/lib/Assist.d.ts
index 1ade011..2d188ae 100644
--- a/node_modules/@openreplay/tracker-assist/lib/Assist.d.ts
+++ b/node_modules/@openreplay/tracker-assist/lib/Assist.d.ts
@@ -2,6 +2,7 @@ import type { Properties } from 'csstype';
 import { App } from '@openreplay/tracker';
 import type { Options as ConfirmOptions } from './ConfirmWindow/defaults.js';
 declare type StartEndCallback = () => ((() => {}) | void);
+// NOTE: check if patch still needed
 export interface Options {
     onAgentConnect: StartEndCallback;
     onCallStart: StartEndCallback;
